README.md

```diff
@@ -1,19 +1,43 @@
 # knuu-example
 
+## Description
+
 This repository contains an example of how to use the [knuu](https://github.com/celestiaorg/knuu) Integration Test Framework.
 
 If you have feedback on the framework, want to report a bug or suggest an improvement, please create an issue in the [knuu](https://github.com/celestiaorg/knuu) repository.
 
+---
 ## Setup
 
+*You need to have access to a Kubernetes cluster.*
+
+### Linux
+
 1. Install required packages
+    
 
     ```shell
     sudo apt-get install buildah pkg-config libgpgme-dev libdevmapper-dev btrfs-progs libbtrfs-dev
     ```
 
+*Install **kubectl**: [install-kubectl-linux](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)*
+
+
 2. Set up access to a Kubernetes cluster using your kubeconfig and create the `test` namespace.
 
+### MacOS
+
+1. Install required packages
+
+    ```shell
+    brew install gpgme
```

Comment:
Is buildah not required for MacOS? Or did you had it installed already?

Reply:
if it's a binary, I don't have it installed:
```bash
zsh: command not found: buildah
```
maybe is not required 🤔